### PR TITLE
feat(gallery): Add 4 tree gallery samples

### PR DESCRIPTION
**Pull request by jbrundage**
Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

### CODE REVIEW

File: demos/gallery/samples/tree/CircleDendrogram.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+data.children[2].children[0].children.push({label:"Ground"});
+data.children[2].children[0].children.push({label:"Rib"});
+data.children[2].children.push({label:"Duck",children:[]});
+data.children[2].children[1].children.push({label:"Leg"});
+data.children[2].children[1].children.push({label:"Breast"});
+data.children[2].children[1].children.push({label:"Thigh"});
+data.children[2].children.push({label:"Chicken",children:[]});
+data.children[2].children[2].children.push({label:"Leg"});
+data.children[2].children[2].children.push({label:"Breast"});
+data.children[2].children[2].children.push({label:"Thigh"});
```
Comment (GordonSmith) [inline demos/gallery/samples/tree/CircleDendrogram.js]: Why not just include the JSON structure?

File: demos/gallery/samples/tree/Dendrogram.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+data.children[2].children[0].children.push({label:"Ground"});
+data.children[2].children[0].children.push({label:"Rib"});
+data.children[2].children.push({label:"Duck",children:[]});
+data.children[2].children[1].children.push({label:"Leg"});
+data.children[2].children[1].children.push({label:"Breast"});
+data.children[2].children[1].children.push({label:"Thigh"});
+data.children[2].children.push({label:"Chicken",children:[]});
+data.children[2].children[2].children.push({label:"Leg"});
+data.children[2].children[2].children.push({label:"Breast"});
+data.children[2].children[2].children.push({label:"Thigh"});
```
Comment (GordonSmith) [inline demos/gallery/samples/tree/Dendrogram.js]: If you export it from the above example you can import it here?

File: demos/gallery/samples/tree/CircleDendrogram.js (hunk 1 above)
Comment (GordonSmith) [reply demos/gallery/samples/tree/CircleDendrogram.js]: And put it below the Widget code in a "createData" method

Comment (jbrundage) [reply demos/gallery/samples/tree/CircleDendrogram.js]: I was just thinking this shows how uniform the input structure is. The JSON seemed to be clearer this way, but at a second glance I guess I was overthinking it...

File: demos/gallery/samples/tree/Dendrogram.js (hunk 2 above)
Comment (jbrundage) [reply demos/gallery/samples/tree/Dendrogram.js]: How would this be done? Like this?
CircleDendrogram.js
```javascript
const treeData = {label:"root",children:[]};
export treeData;
```
Dendrogram.js
```javascript
import treeData from "./CircleDendrogram.js"
```

Comment (GordonSmith) [reply demos/gallery/samples/tree/Dendrogram.js]: Or just 
```
export const treeData = ...
```
Having said that, I am not 100% sure its good to have the sample data in a different file from the users point of view?

**Comment by jbrundage** [comment]
@GordonSmith this is revised

**Pull request merged**